### ISSUE TITLE
Ongoing general review of white paper

### ISSUE THREAD

**Issue by ivilata** [ipfs/papers#11]
Hi, I forked the papers repo and I'm giving the white paper a top-to-bottom review for minor fixes, which include some consistency in wording and formatting, as well as some improvements to the bibliography.  You may see its progress [here](https://github.com/ivilata/papers/tree/misc-fixes).  I'm currently at section _3.3 Routing_.

Should I create a PR for this?  Any other guidelines?

Thanks!

**Comment by jbenet** [comment]
@ivilata thank you for the effort. Some of the changes make sense, some other are stylistic (like the spacing in verbatim segments, etc). would have to see the pdfs to decide on that (i dont have a tex distro atm)-- have a pdf?

In the end DRAFT 3 is due for a major rewrite (DRAFT 4) with _lots_ that will change. Tbh, i dont think it's worth your time to correct much of this as I expect to have a substantially different paper in 2016.

**Comment by ivilata** [comment]
@jbenet I've pushed a commit with the newly generated PDF.

In fact I'm doing the small fixes while rereading the white paper to refresh some IPFS concepts for myself before I start reading the specs for contributing to py-ipfs, so it's not much of an extra effort for me anyway. `:)`  I will continue to upload fixes to this branch, if they are useful for your next draft that would be great, if not it's also ok with me.

Eager to see that new draft!

**Comment by ivilata** [comment]
I completed the review with only very minor style fixes. Maybe the most interesting latest changes are the fixes to bibliography (e.g. for preserving caps) and filling in the missing bibliography entries.

I also commented out the truncated sentence (see #1) and fixed the Kademlia node count example (see #9). The branch includes a rebuilt PDF.

@jbenet, if you haven't yet started DRAFT 4 I can create a PR for the changes to be incorporated, or I can perform further changes if needed. Thanks!

**Comment by jbenet** [comment]
@ivilata i havent started  -- ok feel free to PR. i warn that i may not take all fixes as i have some stylistic idiosyncrasies :)

**Comment by jbenet** [comment]
thanks!

**Comment by ivilata** [comment]
You're welcome! Of course I don't mind if you don't take all changes, telling stylistic decisions apart is not obvious. `;)`

BTW, I amended some late commits which had a wrong directory reference in the comment. PR is #12.

Thanks and good luck with DRAFT 4!

(Moving to specs now…)

**Comment by jbenet** [comment]
@ivilata thank you very much! 

warning on the specs: most is missing, only the network (which is now libp2p) is ample. and even that is not complete / super coherent. any feedback on that one would be great.

and since you read the paper, if you want to contribute to spec writing, it would probably advance the project significantly. I just have way too much on my plate

**Comment by ivilata** [comment]
Yeah, any pointers on which document to tackle first would be very helpful. I'll start with `libp2p` then.

**Comment by axelsimon** [comment]
In `ipfs-p2p-file-system.pdf` DRAFT 3, page 4, in 3.4.2 BitSwap Strategy:
> One choice of function that works in practice is a sigmoid, scaled by a debt retio:

I believe that should be *debt rati**o***.

Also, the “Figure 1: Probability of Sending as r increases” only shows the axes, not any actual plotting of the function.
